README.md

```diff
@@ -187,6 +187,36 @@ the `Intercom` component.
 
 </details>
 
+<details>
+<summary>Messenger</summary>
+
+To use Messenger import the `LiveChatLoaderProvider` and set the `provider` prop
+as `messenger` and the `providerKey` prop as your Facebook App ID.
+
+To use Messenger you have to set two additional props, set the `pageID` prop as your Facebook Page ID and `locale` prop if you prefer any locales. the default locale is `en_US`
+
+Then import the `Messenger` component.
+
+```jsx
+import { LiveChatLoaderProvider, Messenger } from 'react-live-chat-loader'
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <LiveChatLoaderProvider provider="messenger" providerKey="1111222233334444" pageID="111222333444555">
+        /* ... */
+        <Messenger />
+      </LiveChatLoaderProvider>
+    )
+  }
+}
+```
+
+More locale information, 
```

Comment:
For list of `locale` option  value, please refer to [Facebook Localization document](https://developers.facebook.com/docs/internationalization).